api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -471,6 +485,12 @@
      */
     Attribute<ContainerOptConfigurator> CONTAINER_OPTS_CONFIGURATOR = new Attribute<ContainerOptConfigurator>(new Object2String<ContainerOptConfigurator>());
     /**
+     * The strategy to use for stram web services authentication.
+     * See {@link StramHTTPAuthentication}
+     * Default value is StramHTTPAuthentication.FOLLOW_HADOOP_AUTH
+     */
+    Attribute<StramHTTPAuthentication> STRAM_HTTP_AUTHENTICATION = new Attribute<StramHTTPAuthentication>(StramHTTPAuthentication.FOLLOW_HADOOP_AUTH, new StringCodec.Enum2String<>(StramHTTPAuthentication.class));
```

Comment:
Should we take advantage of the diamond operator here: Attribute<> instead of Attribute<StramHTTPAuthentication>